CHANGELOG.md

```diff
@@ -1,5 +1,28 @@
 # Changelog
 
+## 14.1.0 (10/16/23)
+
+### New features
+
+Teleport Connect 14.1 introduces Connect My Computer which makes it possible to add your personal machine to a Teleport cluster in just a couple of clicks. Whether you're exploring capabilities Teleport or want to make your computer available in your private cluster, Connect My Computer lets you do that without having to use the terminal to get the job done. Docs: https://goteleport.com/docs/connect-your-client/teleport-connect/#connect-my-computer
+
+### Other fixes and improvements
+
+* Added desktop access audit events for clipboard and directory sharing, and number of AD/non-AD desktops. [#33455](https://github.com/gravitational/teleport/pull/33455)
+* Added "Connect My Computer" in Teleport Connect for simpler setup. [#33440](https://github.com/gravitational/teleport/pull/33440)
```

Comment:
If there's no single PR which implements the given feature, should we link to the tracking issue instead? (https://github.com/gravitational/teleport/issues/27881)

I originally included the changelog label in #32850 because that was the PR which actually made the feature visible in the app. I see that you decided to link to #33440 instead, so I'm just wondering what would be the best approach overall.

Let me know if I should just link to the tracking issue from PRs with the changelog label in the future.

Reply:
Sorry, I meant to remove that entry from the changelog list of PRs, as it is in the top section. I was not planning on linking to any issues or PRs for these feature additions.